documentation/modules/post/windows/gather/credentials/plsql_developer.md

```diff
@@ -0,0 +1,43 @@
+## Vulnerable Application
+
+This module can decrypt the history of PL/SQL Deceloper, and passwords are available if the user chooses to remember the password.
+
+  Analysis of encryption algorithm [here](https://adamcaudill.com/2016/02/02/plsql-developer-nonexistent-encryption/).
+
+  You can find its official website [here](https://www.allroundautomations.com/products/pl-sql-developer/).
```

Comment:
We have a script that helps tidy up documentation files. If you run the following from the `metasploit-framework` root directory you'll get a couple minor suggestion with regards to formatting:
```
ruby tools/dev/msftidy_docs.rb documentation/modules/post/windows/gather/credentials/plsql_developer.md
```

```suggestion
This module can decrypt the history of PL/SQL Deceloper, and passwords are available if the user chooses to remember the password.
Analysis of encryption algorithm [here](https://adamcaudill.com/2016/02/02/plsql-developer-nonexistent-encryption/).
You can find its official website [here](https://www.allroundautomations.com/products/pl-sql-developer/).
```

Reply:
Thank you for the information! I mistakenly assumed that this was automatically done in the pre-commit hook as well.